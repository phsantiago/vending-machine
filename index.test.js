const calcChange = require('./index.js');

describe('calcChange', () => {
  it('should return a object with kinds of coins availble', () => {
    expect(Object.keys(calcChange()).sort()).toEqual(['1centavo', '5centavos', '10centavos', '25centavos', '50centavos', '1real'].sort());
  })
})
