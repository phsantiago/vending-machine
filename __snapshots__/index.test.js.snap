// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`calcChange should throw exeption when miss fist paramether 1`] = `undefined`;

exports[`calcChange should throw exeption when miss wallet paramether 1`] = `undefined`;

exports[`calcChange should throw exeption when wallet isn't a object 1`] = `undefined`;
